plugins/someteam.example.com/v1/SedTransformer

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+args=""
+
+for arg in $@; do
```

Comment:
This should take the full set of Resources in stdin.

Reply:
it does, that's why TestSedTransformer passes.